tests/data/cases/case_7.json

```diff
@@ -33,18 +33,25 @@
       "startAverageIntegrationStep": 100000
    },
    "surfaceOutput": {
-      "outputFormat": "both",
-      "primitiveVars": true,
-      "Cp": true,
-      "Cf": true,
-      "CfTangent": true,
-      "CfNormal": true,
-      "CfVec": true,
-      "yPlus": true,
-      "wallDistance": true,
-      "Mach": true,
-      "nodeForcesPerUnitArea": true,
-      "animationFrequency": -1
```

Comment:
is this covered by other test?

Reply:
Yes, this boolean legacy pattern is seen in many json files. E.g. case_20.json.
Added unit test for this nonetheless.